setup/monitoring/index.md

```diff
@@ -6,3 +6,262 @@ sidebar:
 ---
 
 {% include toc %}
+Spinnaker is instrumented with numerous metrics internally so that it can be
+monitored. Monitoring spinnaker typically involves the spinnaker-monitoring
+daemon, awhich collects metrics reported by each microservice instance and
+reports them to a third-party monitoring system which you then use to view
+overview dashboards, receive alerts, and informally browse depending on your
+needs.
+
+Spinnaker publishes internal metrics using a multi-dimensional data model
+based on "tags". The metrics, data-model, and usage a discussed further
+in the sections [Consuming Metrics](#consuming-metrics) and in the
+[Monitoring Reference document](/reference/monitoring/).
+
+Spinnaker currently supports three specific third-party systems: 
+[Datadog](https://www.datadoghq.com/), [Prometheus](https://prometheus.io/)
+and [Stackdriver](http://www.stackdriver.com/). The daemon is extensible so
+that it should be straight forward to add other systems as well. In fact,
+each of the supported systems was provided using the daemon's extension
+mechanisms -- there are not "native" systems.
+
+You can also use the microservice HTTP endpoint `/spectator/metrics`
+directly to scrape metrics yourself. The JSON document structure is 
+further documented in the [Monitoring reference section](/reference/monitoring/).
+
+
+## Installing Spinnaker Monitoring (on VMs)
+
+Spinnaker monitoring support is split across two debian packages.
+
+  * `spinnaker-monitoring-daemon` provides the monitoring daemon which is used
+     to collect metrics from one or more microservice instances.
+
+  * `spinnaker-monitoring-third-party` provides the installation scripts and
+    supporting data for the various supported third party services. Installing
+    this package makes the scripts and data available, but does not actually
+    execute them. After installing the package, you can then choose which
+    specific concrete solution(s) to install.
+
+Assuming you have already added the Spinnaker Debian Repositories to your Apt
+Package Manager *TBD - Add reference to instructions*, use the following steps
+to install these packages:
+
+  1. Install the debian packages
+```
+sudo apt-get update -y
+sudo apt-get install spinnaker-monitoring-daemon -y
+sudo apt-get install spinnaker-monitoring-third-party -y
+```
+
+  2. [Configure the Spinnaker Monitoring Daemon](#configure-the-spinnaker-monitoring-daemon)
+
+  3. Install the 3rd-party monitoring package
+     * [Configuring Datadog](/setup/monitoring/datadog)
+     * [Configuring Prometheus](/setup/monitoring/prometheus)
+     * [Configuring Stackdriver](/setup/monitoring/stackdriver)
+
+  4. Restart the Spinnaker Monitoring Daemon
+```
+sudo service spinnaker-monitoring restart
+```
+
+# Configuring the Spinnaker Monitoring Daemon
```

Comment:
On the final page, the title is usually the only thing that gets the `h1` header. I suggest indenting each header with one more hash